README.md

```diff
@@ -58,6 +58,14 @@ An RFC is ready to be approved and finalized once it's Pull Request is ready for
 
 At this time, some member of the core team will motion for a final comment period (FCP). This follows our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process. Once the final comment period has elapsed the RFC will be merged if there are no objections.
 
+## RFC Meetings
+
+RFCs advance through the stages during RFC meetings with TSC and maintainers. Voting follows the [RFC proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
+
+Meetings occur ad hoc rather than on a scheduled basis. They are called for when a proposal author or champion feels it is ready to advance to the next stage. The author or champion can ask for a meeting by contacting TSC to schedule a time.
```

Comment:
```suggestion
Meetings occur ad hoc rather than on a scheduled basis. They are called for when a proposal author or champion feels the proposal is ready to advance to the next stage. The author or champion can ask for a meeting by contacting TSC to schedule a time.
```
"contacting TSC" is pretty vague. Unless we describe how to do that elsewhere on this page, I'd specify *how* they should contact TSC. Do they mention the role in the proposal? Email?

Reply:
I think we should avoid being overly formal. If meetings are ad hoc then there is no formal process. Creating a process makes them now be formal again. So I think spelling it out too much defeats the goal here of making RFC meetings be a loose thing that get called when someone (champion for example) wants to advance an idea.

I don't think this is a real problem in practice, though. The type of people who create RFCs and put in the effort to advance them are part of the community and know how to reach out if they want to see something advance.